coordinator/src/services/translation/cancellation/cancellation-response.ts

```diff
@@ -138,6 +89,37 @@ function createBundleEntries(cancellationResponse: CancellationResponse) {
   return orderedBundleResources.map(convertResourceToBundleEntry)
 }
 
+// TODO - consider building up the list and sorting afterwards, e.g.:
```

Comment:
just a thought, but if were sorting a list, could we also remove duplicates at the same time? might remove some of the logic complication above. i think the comments are useful for understanding where youve added them, maybe add one above the perfomer section too to explain that?

Reply:
Trouble is the Practitioner(Role)s aren't complete duplicates - they'll have different IDs. So if we removed them at the end we'd have to update all the references to point to the new IDs. I think the way you wrote it is the right way to do it.

Reply:
hmmmmmmmmmmmmmmmm okay this may be leaning into it too hard but can we compare equality before we assign IDs? so i guess what i mean is we move the ID generation and assigning of references to right at the end. idk if thats even better at this point lol